src/defaultConfig.js

```diff
@@ -14,7 +14,10 @@ module.exports = function (config) {
     let pathPrefix = pathPrefixOverride || templateConfig.getPathPrefix();
     return urlFilter.call(this, url, pathPrefix);
   });
-  config.addFilter("log", console.log);
+  config.addFilter("log", (input, ...data) => {
```

Comment:
Happy to merge this! Great idea.

Can we rename `data` to `messages` or `logMessages` or something with a bit more readability?

Reply:
Great comment, let's call it messages, since that's more closely to how MDN calls it on console.log ([msg](https://developer.mozilla.org/de/docs/Web/API/Console/log)) and how typescript calls that parameter (messages).

I will update the PR.